src/nspawn/nspawn-register.c

```diff
@@ -41,16 +39,8 @@ static int append_machine_properties(
                                   "char-pts", "rw");
         if (r < 0)
                 return bus_log_create_error(r);
-        if (enable_fuse) {
-                r = sd_bus_message_append(m, "(sv)", "DeviceAllow", "a(ss)", 1,
```

Comment:
This should be kept (merge into _message_append call above?)

Reply:
Ouch, indeed.